kolibri/plugins/coach/assets/src/views/lessons/LessonsRootPage.vue

```diff
@@ -47,7 +47,7 @@
             <th>{{ coreString('progressLabel') }}</th>
             <th>{{ $tr('size') }}</th>
             <th>{{ coachString('recipientsLabel') }}</th>
-            <th>{{ coachString('lessonVisibleLabel') }}</th>
+            <th :id="`toggle-lesson-visibility-label-${lesson.id}`">{{ coachString('lessonVisibleLabel') }}</th>
```

Comment:
`lesson-id` doesn't exist in this scope, since this is a heading. This is a good idea conceptually, but I think with the current table implementation, it won't work, and it's causing an error that is preventing the page from rendering.

For now, please just remove the `-${lesson.id}` here and below, and we can merge it without it. We are currently working on making accessibility improvements to our tables, so we can continue to improve from here :)

Reply:
In the future, we will want to add a more nuanced label -- I think the overall idea you have (putting a unique lesson identifier with the switch) is a good one, but I think it's a bit out of scope of this particular issue. When we work on our updates using our new `KTable` component, we can revisit this labeling. For now, retaining the `toggle-lesson-visibility-label` without the `-${lesson.id}` appended to the end should be sufficient! 👍